src/server/scripts/Kalimdor/BlackfathomDeeps/blackfathom_deeps.cpp

```diff
@@ -238,10 +240,41 @@ class npc_morridune : public CreatureScript
     }
 };
 
+class spell_subjugator_korul_darkness_calls : public SpellScriptLoader    // 151159
+{
+    public:
+        spell_subjugator_korul_darkness_calls() : SpellScriptLoader("spell_subjugator_korul_darkness_calls") { }
+
+        class spell_subjugator_korul_darkness_calls_SpellScript : public SpellScript
+        {
+            PrepareSpellScript(spell_subjugator_korul_darkness_calls_SpellScript);
+
+            void HandleScript(SpellEffIndex /*effIndex*/)
+            {
+                if (Creature* caster = GetCaster()->ToCreature())
```

Comment:
This crashes if the spell is cast by a player. Remove the ToCreature() call since it's not needed at all.